Tests/LogtoMock/NetworkSessionMock.swift

```diff
@@ -27,7 +27,7 @@ public class NetworkSessionMock: NetworkSession {
                         "token_endpoint": "https://logto.dev/token:good",
                         "end_session_endpoint": "https://logto.dev/end:good",
                         "revocation_endpoint": "https://logto.dev/revoke:good",
-                        "userinfo_endpoint": "https://logto.dev/me:good",
+                        "userinfo_endpoint": "https://logto.dev/user",
```

Comment:
wondering is anyone referencing it?

Reply:
just test cases

Reply:
https://github.com/logto-io/swift/pull/52/files#diff-97bcf0f08aaa8e839d22785c49f2e71ce839e8a33eea04ea00c80fca5a27e3d0R47-R57